go/runtime/host/sandbox/sandbox.go

```diff
@@ -0,0 +1,60 @@
+// Package sandbox implements a process sandboxing mechanism.
```

Comment:
It might be nicer moving `Process` (and it's implementations naked & bwrap) into `go/runtime/host/sandbox/process` package

Reply:
Good idea.